assets/models/system/compvis-stable-diffusion-v1-4/spec.yaml

```diff
@@ -3,15 +3,31 @@ name: compvis-stable-diffusion-v1-4
 path: ./
 properties:
   SHA: 133a221b8aa7292a167afc5127cb63fb5005638b
-  datasets: LAION-5B
-  inference-min-sku-spec: 4|1|28|176
+  inference-min-sku-spec: 6|1|112|336
```

Comment:
why this change?

Reply:
this was error in old spec, corrected it. Added min sku configuration of nc6.